frontend/src/components/pages/books/BookList.js

```diff
@@ -0,0 +1,41 @@
+import { useState } from "react";
+
+import { Grid, Typography } from "@mui/material";
+import { useQuery } from "@tanstack/react-query";
+
+import BookListItem from "./BookListItem";
+import SearchField from "../../ui/SearchField";
+
+const BookList = ({ fetchBooks }) => {
+  const [searchQuery, setSearchQuery] = useState();
+
+  const { data: rawBooks, isSuccess } = useQuery(
+    ["books", searchQuery],
+    fetchBooks,
+    {
+      retry: 10,
+    }
+  );
+
+  return (
+    <>
+      <Typography variant="h3">Book Search</Typography>
+      <SearchField
+        placeholder="e.g. Philosopher's Stone"
+        handleChangeSearch={setSearchQuery}
+        totalResults={rawBooks?.pages
+            ? rawBooks?.meta?.pagination?.records
+            : 0}
```

Comment:
```suggestion
        totalResults={rawBooks?.meta?.pagination?.records || 0}
```
This should fix the results count.